python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
By that logic all of these should be moved to Ganga as they do nothing unique to DIRAC and worse require special treatment due to the fact that DIRAC has extra requirements external to these tests. I don't know why they've been placed here I was assuming there was a good reason.